pkg/identity/certificate_authority.go

```diff
@@ -95,21 +96,33 @@ func NewCA(ctx context.Context, opts NewCAOptions) (_ *FullCertificateAuthority,
 		opts.Concurrency = 1
 	}
 
-	log.Printf("Generating a certificate matching a difficulty of %d\n", opts.Difficulty)
+	fmt.Printf("Generating key with a minimum a difficulty of %d...\n", opts.Difficulty)
+	logStatus := func() {
+		count := atomic.LoadUint32(i)
+		hs := atomic.LoadUint32(highscore)
+		fmt.Printf("\rGenerated %d keys; best difficulty so far: %d", count, hs)
+	}
 	err = GenerateKeys(ctx, minimumLoggableDifficulty, int(opts.Concurrency),
 		func(k *ecdsa.PrivateKey, id storj.NodeID) (done bool, err error) {
+			count := atomic.LoadUint32(i)
```

Comment:
Is this awful for performance?

Reply:
Use `count := atomic.AddUint32(i, 1)` and it's ~10ns per call.... i.e. the modulo operator can be slower.

Reply:
* it's lower when there is no contention

Reply:
ha - that makes sense